snuba/utils/streams/backends/kafka.py

```diff
@@ -629,6 +629,16 @@ def closed(self) -> bool:
     "sasl.username",
     "sasl.password",
     "security.protocol",
+    "ssl.client.auth",
+    "ssl.endpoint.identification.algorithm",
+    "ssl.key.location",
+    "ssl.key.password",
+    "ssl.key.pem",
+    "ssl.keystore.location",
+    "ssl.keystore.password",
+    "ssl.truststore.location",
+    "ssl.truststore.password",
+    "ssl_key",
```

Comment:
What's the type of this setting according to the python api?  If it is not a string it likely won't work as it is.

Reply:
Removed this one as it's not necessarily needed to configure SSL.

Reply:
Ok, have removed it.